docs/architecture/self-hosted/authentication/databricks-oauth.md

```diff
@@ -20,6 +20,8 @@ Databricks supports the following OAuth use cases:
 - Works with Spark (clusters) as well as SQL (warehouses).
 - Works with one or more Prophecy Fabrics.
   - Allow users to quickly establish multiple Prophecy Fabrics, each linked to a separate Databricks schema, leveraging the same OAuth connection/tokens.
+- Supports authentication with Service Principals, allowing Prophecy to perform release tasks without requiring user interaction.
+- Enables fields for OAuth configuration (Service Principal Client ID and Client secret) when creating Fabrics, supporting environments that need automated administration.
```

Comment:
Shouldn't one of these 2 points be about U2M authentication for interactive flows?

Reply:
Could you elaborate on this?